src/quacc/recipes/espresso/core.py

```diff
@@ -148,6 +152,91 @@ def relax_job(
     )
 
 
+@job
+def external_relax_job(
+    atoms: Atoms | UnitCellFilter,
+    preset: str | None = "sssp_1.3.0_pbe_efficiency",
+    parallel_info: dict[str] | None = None,
+    test_run: bool = False,
+    opt_params: dict[str, Any] | None = None,
+    opt_patch: Any | None = None,
+    copy_files: str | Path | list[str | Path] | None = None,
+    **calc_kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a structure relaxation with pw.x.
+
+    Parameters
+    ----------
+    atoms
+        The Atoms object or the result of a UnitCellFilter for cell-relaxation.
+    preset
+        The name of a YAML file containing a list of parameters to use as
+        a "preset" for the calculator. quacc will automatically look in the
+        `ESPRESSO_PRESET_DIR` (default: quacc/calculators/espresso/presets).
+    relax_cell
+        Whether to relax the cell or not.
+    parallel_info
+        Dictionary containing information about the parallelization of the
+        calculation. See the ASE documentation for more information.
+    test_run
+        If True, a test run is performed to check that the calculation input_data is correct or
+        to generate some files/info if needed.
+    copy_files
+        List of files to copy to the calculation directory. Useful for copying
+        files from a previous calculation. This parameter can either be a string
+        or a list of strings.
+
+        If a string is provided, it is assumed to be a path to a directory,
+        all of the child tree structure of that directory is going to be copied to the
+        scratch of this calculation. For phonon_job this is what most users will want to do.
+
+        If a list of strings is provided, each string point to a specific file. In this case
+        it is important to note that no directory structure is going to be copied, everything
+        is copied at the root of the temporary directory.
+    **calc_kwargs
+        Additional keyword arguments to pass to the Espresso calculator. See the
+        docstring of [quacc.calculators.espresso.espresso.Espresso][] for more information.
+
+    Returns
+    -------
+    RunSchema
+        Dictionary of results from [quacc.schemas.ase.summarize_run][].
+        See the type-hint for the data structure.
+    """
+
+    calc_defaults = {
+        "input_data": {
+            "control": {
+                "calculation": "scf",
+                "tstress": True if isinstance(atoms, UnitCellFilter) else False,
+                "tprnfor": True,
+            }
+        }
+    }
+
+    opt_defaults = {
+        "fmax": 0.01,
+        "max_steps": 500,
+        # LBFGS is very often the "best" optimizer for the general case.
+        "optimizer": LBFGS
```

Comment:
Elsewhere I had been using max_steps of 1000 and FIRE for the optimizer. But I don't think it really matters much in practice.